build.sbt

```diff
@@ -186,9 +186,19 @@ lazy val core = project
       catsEffect.value,
       mamlJvm,
       simulacrum,
-      scalatest
+      scalatest,
+      droste
     )
   )
+  .settings(
+    libraryDependencies ++= {
+      CrossVersion.partialVersion(scalaVersion.value) match {
+        case Some((2, scalaMajor)) if scalaMajor == 11 =>
+          Seq(circeJava8.value)
```

Comment:
In Scala 2.12 java8.time codecs are built in, that is why we have a compact object